src/components/common/SearchBar/searchBar.tsx

```diff
@@ -332,15 +334,30 @@ const SearchBar = ({
           );
         }}
       />
-      <Button
-        variant="contained"
-        disableElevation
-        size="large"
-        className="shrink-0 normal-case bg-royal hover:bg-royalDark"
-        onClick={() => onSelect_internal(value)}
+      <Tooltip
+        title="Select an option before searching"
+        placement="top"
+        open={openErrorTooltip}
+        onOpen={() => setErrorTooltip(true)}
+        onClose={() => setErrorTooltip(false)}
+        disableFocusListener
+        disableHoverListener
+        disableTouchListener
       >
-        Search
-      </Button>
+        <Button
+          variant="contained"
+          disableElevation
+          size="large"
+          className={
+            value.length == 0
+              ? 'shrink-0 normal-case bg-royal hover:bg-royalDark text-cornflower-100'
+              : 'shrink-0 normal-case bg-royal hover:bg-royalDark'
```

Comment:
can this be:

`'shrink-0 normal-case bg-royal hover:bg-royalDark' + (value.length == 0 ? ' text-cornflower-100' : '')`

so it's more obvious what's changing